packs/feats/ostilli-host-dedication.json

```diff
@@ -11,7 +11,7 @@
+            "value": "<p><strong>Access</strong> surki ancestry</p><hr /><p>You've bonded with an attached symbiote known as an ostilli. You can bond with only one ostilli at a time since the symbiote emits low-frequency magical pulses that repel other ostillis. You become trained in Ostilli Lore; if you were already trained, you become an expert.</p>\n<p>The ostilli is a Tiny creature grafted to your body. Like other grafts, it has no Hit Points or Speeds of its own and can't be targeted separately. It can't be removed and dies when you do; in the event of your demise and resurrection, you can bond to a new ostilli during a week of downtime, though you lose any abilities granted by your ostilli bond during that time. Your ostilli is obvious, unless you attempt to cover it with clothing or armor. In such a case, an onlooker can determine you're bonded to an ostilli with a successful Nature, Perception, or Surki Lore check against your Deception DC. Your ostilli must be visible for you to use any of the actions it grants.</p>\n<p>Your ostilli is constantly siphoning ambient magic from the surroundings, granting you the Repel Ambient Magic and Spit Ambient Magic actions.</p>\n<p><strong>Repel Ambient Magic</strong> <span class=\"action-glyph\">1</span> (concentrate) Your ostilli glows green as it absorbs magic. Until the start of your next turn, you gain a +1 circumstance bonus to AC and saving throws against the next magical attack, cantrip, or spell that targets you. This bonus increases to +2 at 12th level.</p>\n<p><strong>Spit Ambient Magic</strong> <span class=\"action-glyph\">1</span> (concentrate, magical)</p>\n<p><strong>Frequency</strong> once per round</p><hr /><p><strong>Effect</strong> Your ostilli flashes red as it regurgitates a dart of magic at a target within 30 feet. This magical dart deals @Damage[1d6[piercing]] damage (basic Reflex save against the higher of your class DC or spell DC). This damage increases by 1d6 at 6th level and every 4 levels thereafter.</p>"
```

Comment:
```suggestion
            "value": "<p><strong>Access</strong> surki ancestry</p><hr /><p>You've bonded with an attached symbiote known as an ostilli. You can bond with only one ostilli at a time since the symbiote emits low-frequency magical pulses that repel other ostillis. You become trained in Ostilli Lore; if you were already trained, you become an expert.</p>\n<p>The ostilli is a Tiny creature grafted to your body. Like other grafts, it has no Hit Points or Speeds of its own and can't be targeted separately. It can't be removed and dies when you do; in the event of your demise and resurrection, you can bond to a new ostilli during a week of downtime, though you lose any abilities granted by your ostilli bond during that time. Your ostilli is obvious, unless you attempt to cover it with clothing or armor. In such a case, an onlooker can determine you're bonded to an ostilli with a successful Nature, Perception, or Surki Lore check against your Deception DC. Your ostilli must be visible for you to use any of the actions it grants.</p>\n<p>Your ostilli is constantly siphoning ambient magic from the surroundings, granting you the @UUID[Compendium.pf2e.actionspf2e.Item.Repel Ambient Magic] and @UUID[Compendium.pf2e.actionspf2e.Item.Spit Ambient Magic] actions.</p>"
```

Just link to the actions now that they exist.

Can't make a suggestion on the other feats since the description wasn't touched but similar story there.